src/components/Navbar/SiteNavigation.tsx

```diff
@@ -43,11 +43,7 @@ export default function SiteNavigation() {
   return (
     // top-nav is antd override
     <nav className="top-nav z-[1] flex h-16 items-center justify-between bg-smoke-25 py-2 px-12 leading-[64px] dark:bg-slate-800">
-      <Menu
-        className="flex w-[500px] flex-row"
```

Comment:
Do we need to specify the width of a menu here? I've removed it since it's flex but I can add '600px' for example if we need it.

With adding a new link to our navbar, on small laptops and tablets it's a bit cumbersome, we can probably think about shrinking it better in some near future ( gonna need designer input probably )

Reply:
Great point. I think you made the right call to remove it.

**suggestion**: WDYT about hiding the `Create a project` button on screens less than tailwind's `lg` (1024px)? I played around with it and i think it might be okay.

Reply:
SGTM, it's a good quick fix. I've applied the tailwind class to element in desktop items and removed it from mobile.